builder/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	runner := buildpackrunner.New(&config)
 
-	err := runner.Run()
+	_, err := runner.Run()
```

Comment:
I don't have super strong feelings, but I do think it is not clear that the `buildpackrunner` is intended to be used as a library in a separate location. Currently there are no unit tests and there is one location in the repository that calls this method and subsequently ignores it. I could easily see us making a change to simplify this and being unaware that we are breaking another team. Any thoughts on how we can improve this?